.github/workflows/cross-version-tests.yml

```diff
@@ -78,7 +78,8 @@ jobs:
           else
             python_version=3.6
           fi
-          echo "::set-output name=python-version::$python_version"
+          latest_micro_version=$($CONDA/bin/conda search python | grep $python_version | tail -1 | tr -s ' ' | cut -d' ' -f2)
+          echo "::set-output name=python-version::$latest_micro_version"
```

Comment:
should this be:
```
echo "::set-output name=python-version::${python_version}.$latest_micro_version"
```

Reply:
Offline discussion: we agreed on the current code and variable name.